js/about/preferences.js

```diff
@@ -260,9 +260,20 @@ class BitcoinDashboard extends ImmutableComponent {
   get ledgerData () {
     return this.props.ledgerData
   }
-  get overlayContent () {
+  get bitcoinOverlayContent () {
     return <iframe src={this.ledgerData.get('buyURL')} />
   }
+  get qrcodeOverlayContent () {
+    return <div>
+      <img src={this.ledgerData.get('paymentIMG')} title='Brave wallet QR code' />
+      <div className='bitcoinQRTitle' data-l10n-id='bitcoinQR' />
+    </div>
+  }
+  get qrcodeOverlayFooter () {
```

Comment:
@bradleyrichter how does that sound to you? I was just going off your mock https://cloud.githubusercontent.com/assets/13509546/18232727/0a78554c-728b-11e6-9146-e8196a95e39d.png